en/docs/design/create-api/test-a-rest-api.md

```diff
@@ -19,22 +19,25 @@ Follow the instructions below to use the Publisher test console to test an API:
 
 3.  Click **TryOut**.
 
-     <a href="{{base_path}}/assets/img/design/create-api/test-api/publisher-testconsole-leftpane.png"><img src="{{base_path}}/assets/img/design/create-api/test-api/publisher-testconsole-leftpane.png" width="20%" alt="Try out menu option in the left panel"></a>
-
-4.  Click **GET TEST KEY**.
-
-     This generates the internal key needed to invoke the API.
-
+     <html><div class="admonition note">
+      <p class="admonition-title">Note</p>
+      <ul> The API can be tested in any lifecycle state, and can be used in any state except for retired, even in block
```

Comment:
@hasuniea , please update this message after trying this out as discussed offline.

Reply:
fixed